command_line/show.py

```diff
@@ -299,6 +307,35 @@ def show_experiments(experiments, show_scan_varying=False, show_image_statistics
     return "\n".join(text)
 
 
+def model_connectivity(experiments):
+    def model_connectivity_impl(experiments, model):
+        text = [""]
+        text.append("%s:" % model.capitalize())
+        models = getattr(experiments, "%ss" % model)()
+        rows = [[""] + [str(j) for j in range(len(models))]]
+        for j, e in enumerate(experiments):
+            row = ["Experiment %d" % j]
+            for m in models:
+                if getattr(e, model) is m:
+                    row.append("X")
+                else:
+                    row.append("/")
```

Comment:
```suggestion
                    row.append(".")
```

my :bikeshed: contribution.